ui/components/common/CodeEditor.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component, createRef } from "react";
+
+import style from "./codeEditor.css";
```

Comment:
@cfoskin @josemigallas what do you think about using styles in this way? (Requires a slight modification to the webpack config).

Reply:
@pb82 👍  no issues with it